samples/automanage/README.md

```diff
@@ -0,0 +1,134 @@
+            "LogAnalytics/Enable": True,
+            "BootDiagnostics/Enable": True
+        }
+    }
+}
+
+client.configuration_profiles.create_or_update("configurationProfileName", "resourceGroupName", new_profile)
+ ```
+
+ #### Get an Automanage Configuration Profile
+
+ ```python
+profile = client.configuration_profiles.get("configurationProfileName", "resourceGroupName")
+ ```
+
+#### Delete an Automanage Configuration Profile
+
+ ```python
+client.configuration_profiles.delete("resourceGroupName", "configurationProfileName")
+ ```
+
+#### Get an Automanage Profile Assignment
+
+```python 
+assignment = client.configuration_profile_assignments.get("resourceGroupName", "default", "vmName")
+```
+
+#### Create an Assignment between a VM and an Automanage Best Practices Production Configuration Profile
+ 
+The **Best Practices Profile** live at the tenant scope, so the **configurationProfile** path would be: `/providers/Microsoft.Automanage/bestPractices/AzureBestPracticesProduction`
+
+ ```python 
+assignment = {
+    "properties": {
+        "configurationProfile": "/providers/Microsoft.Automanage/bestPractices/AzureBestPracticesProduction",
+    }
+}
+
+# assignment name must be named 'default'
+client.configuration_profile_assignments.create_or_update("default", "resourceGroupName", "vmName", assignment)
+ ```
+
+#### Create an Assignment between a VM and a Custom Automanage Configuration Profile
+
+**Custom Profiles** live within resource groups, so the **configurationProfile** path would be: `/subscriptions/<sub ID>/resourceGroups/resourceGroupName/providers/Microsoft.Automanage/configurationProfiles/configurationProfileName`
+
+
+```python
+assignment = {
+    "properties": {
+        "configurationProfile": "/subscriptions/<sub ID>/resourceGroups/resourceGroupName/providers/Microsoft.Automanage/configurationProfiles/configurationProfileName"
+    }
+}
+
+# assignment name must be named 'default'
+client.configuration_profile_assignments.create_or_update("default", "resourceGroupName", "vmName", assignment)
+
+```
+
+To learn how to use this package, see the [quickstart guide](https://aka.ms/azsdk/python/mgmt)
+ 
+For docs and references, see [Python SDK References](https://docs.microsoft.com/python/api/overview/azure/)
+Code samples for this package can be found at [Auto Manage Management](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com.
+Additional code samples for different Azure services are available at [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
+
+
+# Provide Feedback
+
+If you encounter any bugs or have suggestions, please file an issue in the
+[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
+section of the project. 
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fazure-mgmt-automanage%2FREADME.png)
```

Comment:
It is recommended to replace the entire README.md file content, so as to comply with the README.md specification of the main library.

```suggestion
---
page_type: sample
languages:
- python
products:
- azure
description: "These code samples will show you how to manage Automanage using Azure SDK for Python."
urlFragment: Automanage
---

# Getting started - Managing Automanage using Azure Python SDK

These code samples will show you how to manage Automanage using Azure SDK for Python.

## Features

This project framework provides examples for the following services:

### Automanage
* [] Using the Azure SDK for Python - Automanage Management Library [azure-mgmt-automanage](https://pypi.org/project/azure-mgmt-automanage/) for the [Automanage API](https://docs.microsoft.com/en-us/rest/api/automanage/)

## Getting Started

### Prerequisites

1. Before we run the samples, we need to make sure we have setup the credentials. Follow the instructions in [register a new application using Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to obtain `subscription id`,`client id`,`client secret`, and `application id`

2. Store your credentials an environment variables.
For example, in Linux-based OS, you can do
```bash
export AZURE_TENANT_ID="xxx"
export AZURE_CLIENT_ID="xxx"
export AZURE_CLIENT_SECRET="xxx"
export SUBSCRIPTION_ID="xxx"
```

### Installation

1.  If you don't already have it, [install Python](https://www.python.org/downloads/).

    This sample (and the SDK) is compatible with Python 3.7+.

2.  General recommendation for Python development is to use a Virtual Environment.
    For more information, see https://docs.python.org/3/tutorial/venv.html

    Install and initialize the virtual environment with the "venv" module on Python 3 (you must install [virtualenv](https://pypi.python.org/pypi/virtualenv) for Python 2.7):

    ```
    python -m venv mytestenv # Might be "python3" or "py -3.6" depending on your Python installation
    cd mytestenv
    source bin/activate      # Linux shell (Bash, ZSH, etc.) only
    ./scripts/activate       # PowerShell only
    ./scripts/activate.bat   # Windows CMD only
    ```

### Quickstart

1.  Clone the repository.

    ```
    git clone https://github.com/Azure-Samples/azure-samples-python-management.git
    ```

2.  Install the dependencies using pip.

    ```
    cd azure-samples-python-management/samples/automanage
    pip install -r requirements.txt
    ```

## Demo

A demo app is included to show how to use the project.

To run the complete demo, execute `python example.py`

To run each individual demo, point directly to the file. For example (i.e. not complete list):

1. `python manage_private_endpoint_list.py`

If the script starts with `disable_***.py`, it means that it is unavailable now.

The sample files do not have dependency each other and each file represents an individual end-to-end scenario. Please look at the sample that contains the scenario you are interested in

## Resources

- https://github.com/Azure/azure-sdk-for-python

```